src/utils/consts.js

```diff
@@ -10,6 +10,23 @@ const pages = {
 
 const consts = {
   baseUrl: process.env.BASE_URL ?? "http://localhost:3000",
+
```

Comment:
Could you also add a `tagsArray` here with values `["auditory", "heartworm", "fleas/ticks"]`?